include/Smarty/plugins/modifier.htmlparser.php

```diff
@@ -0,0 +1,23 @@
+<?php
+// NOTE => Used to show content in Detail View
+/**
+ * Smarty plugin
+ * @package Smarty
+ * @subpackage plugins
+ */
+
+/**
+ * Smarty HTML parser modifier plugin
+ *
+ * Type:     modifier<br>
+ * Name:     htmlparser<br>
+ * Purpose:  parse html to display
+ * @param string
+ * @return string
+ */
+function smarty_modifier_htmlparser($string, $uc_digits = false)
```

Comment:
You are not using the variable `$uc_digits` in the function.

Reply:
Thanks, I have just committed the changes.